packages/terra-list/src/ListItem.jsx

```diff
@@ -118,9 +118,11 @@ const ListItem = ({
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
   if (isDraggable) {
-    attrSpread.onClick = (event) => {
-      event?.currentTarget?.focus();
-    };
+    if (!onSelect) {
+      attrSpread.onClick = (event) => {
+        event?.currentTarget?.focus();
```

Comment:
what if this line is move here: https://github.com/cerner/terra-core/blob/4af794a10d19faddf2880e0523b779a84a5ef263/packages/terra-list/src/ListUtils.js#L28